server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/GetSnapshotsResponse.java

```diff
@@ -62,10 +72,22 @@
     @Nullable
     private final String next;
 
-    public GetSnapshotsResponse(List<SnapshotInfo> snapshots, Map<String, ElasticsearchException> failures, @Nullable String next) {
+    private final int totalCount;
```

Comment:
nit: I would prefer to just call this `total` to keep it in line with both the `remaining` field (which should otherwise also nit-wise be called `remainingCount`) and the REST layer. But this is ofc optional if you prefer it like it is...